test/extensions/ostree_repository_test.rb

```diff
@@ -0,0 +1,118 @@
+# Copyright 2014 Red Hat, Inc.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+# LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+# WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+require 'rubygems'
+require 'minitest/autorun'
+
+require './test/support/repository_support'
+require './lib/runcible'
+
+module Extensions
+  module TestOstreeRepositoryBase
+    def setup
+      @support = RepositorySupport.new('ostree')
+      @extension = TestRuncible.server.extensions.repository
+    end
+  end
+
+  class TestOstreeRepositoryCreate < MiniTest::Unit::TestCase
+    include TestOstreeRepositoryBase
```

Comment:
@ehelms Just following the template we are using for other kind of repository tests. Who know we may use it to do sync tests